legal-api/src/legal_api/models/review.py

```diff
@@ -94,7 +94,8 @@ def get_paginated_reviews(cls, page, limit):
         for review, effective_date in results:
             future_effective_date = ''
             if effective_date > datetime.now(timezone.utc):
-                future_effective_date = LegislationDatetime.format_as_legislation_date(effective_date)
+                effective_date_utc = LegislationDatetime.as_utc_timezone(effective_date)
+                future_effective_date = effective_date_utc.isoformat()
```

Comment:
effective_date is already in utc timezone
```suggestion
                future_effective_date = effective_date.isoformat()
```

Reply:
Oops... I am definitely not good working with dates :)